diff --git a/android/build.gradle b/android/build.gradle
--- a/android/build.gradle
+++ b/android/build.gradle
@@ -12,7 +12,7 @@
 }
 
 android {
-    compileSdkVersion safeExtGet('compileSdkVersion', 31)
+    compileSdk safeExtGet('compileSdkVersion', 31)
 
     // Conditional for compatibility with AGP <4.2.
     if (project.android.hasProperty("namespace")) {
@@ -27,8 +27,8 @@
     }
 
     defaultConfig {
-        minSdkVersion safeExtGet('minSdkVersion', 21)
-        targetSdkVersion safeExtGet('targetSdkVersion', 31)
+        minSdk safeExtGet('minSdkVersion', 21)
+        targetSdk safeExtGet('targetSdkVersion', 31)
         versionCode 1
         versionName "1.0"
         ndk {
