diff --git a/android/build.gradle b/android/build.gradle
--- a/android/build.gradle
+++ b/android/build.gradle
@@ -133,7 +133,7 @@
     compose shouldIncludeCompose
   }
 
-  packagingOptions {
+  packaging {
     // Gradle will add cmake target dependencies into packaging.
     // Theses files are intermediated linking files to build modules-core and should not be in final package.
     def sharedLibraries = [
